### PR TITLE
Add .PHONY: to make rules

### PR DESCRIPTION
**Pull request by roytman**
Add  .PHONY: to make rules

Signed-off-by: Alexey Roytman <roytman@il.ibm.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -40,6 +40,7 @@ deploy: $(TOOLBIN)/kubectl $(TOOLBIN)/helm
 	$(TOOLBIN)/helm install fybrik charts/fybrik --values $(VALUES_FILE) \
                --namespace $(KUBE_NAMESPACE) --wait --timeout 120s
 
+.PHONY: pre-test
```
Comment (shlomitk1) [inline Makefile:43]: According to https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html, 
`a phony target should not be a prerequisite of a real target file`.
`pre-test` is a prerequisite of `test`

**Comment by shlomitk1** [comment]
Is there an issue this PR is related to? Why is it required?

**Comment by roytman** [comment]
> 



> Is there an issue this PR is related to? Why is it required?

No, I did not open an issue, but in Makefiles all targets that are not connected to real files need the `.PHONY` annotation. https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

File: Makefile (hunk 1 above)
Comment (shlomitk1) [reply Makefile:43]: @roytman ^^^

**Review by shlomitk1 — APPROVED** [review]

**Pull request merged**